hoot-core/src/main/cpp/hoot/core/io/HootApiDbReader.cpp

```diff
@@ -97,7 +97,7 @@ void HootApiDbReader::open(QString urlStr)
     if (_email == "")
     {
       throw HootException("If a map name is specified then the user email must also be specified "
-                          "via: " + emailKey());
+                          "via: " + ConfigOptions().getApiDbEmailKey());
```

Comment:
No, that's a good point...I'll change it.

The ServiceDbEmptyTagsTest failure on the last go around is a little odd.  I got that failure once when running locally after making these changes, but then I couldn't reproduce it again...but jenkins did.  I'll have to figure out what's going on there.